src/realm/utilities.hpp

```diff
@@ -361,6 +361,14 @@ constexpr inline size_t round_down(size_t p, size_t align)
     return r & (~(align - 1));
 }
 
+// return pointer to found character or to terminating NUL
+static inline const char* find_chr(const char* p, char c)
```

Comment:
Why do we need this utility? It seems to me that it is just a `char *strchr(const char *string, int c);`

Reply:
strchr will return nullptr is not found. We need it to return a pointer to terminating NUL character. GCC has a function called strchrnul, but that is not generally available.

Reply:
OK, I think it does not make a huge difference if we call `strchr()` and we check for nullptr in order to return a null terminated string or loop over like we do.